src/components/avatar/Avatar.css

```diff
@@ -0,0 +1,64 @@
+:root{
+    --dark-blue:rgb(29, 29, 46);
+    --gray:#999999;
+    --dark-green:rgb(49, 94, 75);
+    --yellow:rgb(174, 177, 42);
+    --pink:rgb(241, 136, 201);
+}
+.demo-container{
+    background-color:rgb(235, 233, 233);
+    padding:2rem;
```

Comment:
Use variables instead of direct values, if only used once mention it using comment.